test/integration/component/test_ldap_autoImport.py

```diff
@@ -0,0 +1,663 @@
+
+            cls.debug("LDAP Configuration failed with exception")
+
+        if cls.ldapconfRes == 1:
+            global class1
+            class1 = 1
+        cls.cleanup.append(cls.ldapconfRes)
+
+    @classmethod
+    def tearDownClass(cls):
+        #delete ldap configuration
+        """
+        #cleanup includes : delete normal account, remove ldap configuration
+        :type cls: object
+
+        try:
+            cleanup_resources(cls.api_client, cls.cleanup)
+
+        except Exception as tde:
+            raise Exception("Warning: Exception during cleanup : %s" % tde)
+        return
+        """
+        testClient = super(TestLdap1, cls).getClsTestClient()
+        cls.api_client = testClient.getApiClient()
+        cls.services = testClient.getParsedTestDataConfig()
+        try:
+                deleteAcct = deleteAccount.deleteAccountCmd()
+                deleteAcct.id = cls.acctRes.id
+
+                acct_name = cls.acctRes.name
+
+                cls.apiClient.deleteAccount(deleteAcct)
+
+                cls.debug(
+                    "Deleted the the following account name %s:" %
+                    acct_name)
+
+        except Exception as e:
+                raise Exception("Warning: Exception during cleanup : %s" % e)
+
+
+        if cls.ldapconfRes == 1:
+                    ldapserver = deleteLdapConfiguration.deleteLdapConfigurationCmd()
+                    ldapserver.hostname =  cls.services["configurableData"]["ldap_configuration"]["hostname"]
+
+                    try:
+                        cls.apiClient.deleteLdapConfiguration(ldapserver)
+                        cls.debug("deleteLdapConfiguration was successful")
+                        return 1
+                    except Exception as e:
+                        cls.debug("deleteLdapConfiguration failed %s" % e)
+                        return 0
+
+
+
+    def setUp(self):
+
+
+        #create a domain
+        #link ldap to domain
+        self.delflag  = 0
+        self.delflag2 = 0
+        self.delflag3 = 0
+        self.delflag4 = 0
+        self.delflag5 = 0
+        self.delflag6 = 0
+
+        self.assertEquals(
+            class1,
+            1,
+            self.reason)
+        self.apiclient = self.testClient.getApiClient()
+        self.dbclient = self.testClient.getDbConnection()
+        self.cleanup = []
+
+
+
+        self.domain = createDomain.createDomainCmd()
+        self.domain.name = self.services["configurableData"]["link_ldap_details"]["domain_name"]
+
+        if self.domain.name == "":
+            self.debug("Please rerun the test by providing values in ldap configuration user details")
+            self.skipTest("Please rerun the test by providing proper values in configuration file")
+
+        else:
+            self.delflag = 1
+            self.domainRes = self.apiClient.createDomain(self.domain)
+        self.assertEquals(self.delflag, 1, "Domain creation failed,please check the configuration")
+        #return
```

Comment:
Please remove code, do not comment it. It can be retrieved if necessary from Git.